tools/link.py

```diff
@@ -216,8 +216,7 @@ def get_js_sym_info():
   # and can contain full paths to temporary files.
   skip_settings = {'PRE_JS_FILES', 'POST_JS_FILES'}
   input_files = [json.dumps(settings.external_dict(skip_keys=skip_settings), sort_keys=True, indent=2)]
-  for jslib in sorted(glob.glob(utils.path_from_root('src') + '/library*.js')):
-    input_files.append(read_file(jslib))
+  input_files.extend(read_file(jslib) for jslib in sorted(glob.glob(utils.path_from_root('src') + '/library*.js')))
```

Comment:
Is there a way to configure it to avoid generating overly complex list comprehension like this?    I think there is too much going on here for a single line of code.

Reply:
```suggestion
  jslibs = glob.glob(utils.path_from_root('src') + '/library*.js')
  input_files.extend(read_file(jslib) for jslib in sorted(jslibs))
```